Update icon for new branding

ryanjduffy opened the pull request:
Updates icon assets

![image](https://user-images.githubusercontent.com/788456/132109444-310eb0ae-cf10-48bd-a7e0-6ce0278e79cd.png)
![image](https://user-images.githubusercontent.com/788456/132074803-42c8dc47-867c-4b6d-b45d-4d8df2f1e4ec.png)

ryanjduffy committed:
Update icon for new branding

jasonLaster commented:
This is awesome!

Could you add some docs on how you updated this? Where did the icons come from for instance?

ryanjduffy commented:
Fixed these up today and moved the assets into the Replay Design Doc: figma.com/file/SSr1ljyzF0lCXTL7BX1Whh/Replay-Design-Doc?node-id=6214%3A30763

I lifted the icons and background gradients/colors from the website design and then used an icns/ico plugin for figma to export them with the proper frame so the icon wasn't oversized.

ryanjduffy commented:
The drop shadow doesn't look quite right to me but this is closer overall.

![image](https://user-images.githubusercontent.com/788456/132109444-310eb0ae-cf10-48bd-a7e0-6ce0278e79cd.png)

ryanjduffy committed:
fix icon size in frame
